pkg/addons/data/Deployment.apps/default/registry-console.yaml

```diff
@@ -48,5 +48,10 @@ spec:
             port: 9090
             scheme: HTTP
           timeoutSeconds: 5
+      dnsPolicy: ClusterFirst
       nodeSelector:
-        region: infra
+        node-role.kubernetes.io/master: "true"
```

Comment:
Why master?

Reply:
Cool point, again default in upstream. Will change.

Reply:
move to infra